modules/ROOT/partials/configuration/autocorrect_api_key.adoc

```diff
@@ -0,0 +1,17 @@
+[[autocorrect_api_key]]
```

Comment:
This is not included anywhere... that said, this is an internal option and like all other premium plugins isn't something that should be documented at this stage. So please remove this as well.